core/server/master/src/main/java/alluxio/master/file/meta/UfsSyncUtils.java

```diff
@@ -110,13 +130,15 @@ public static boolean inodeUfsIsSynced(Inode inode, String ufsFingerprint) {
    */
   public static final class SyncPlan {
     private boolean mUpdateDirectory;
+    private boolean mUpdateMetadata;
```

Comment:
can we just have `mUpdateMetadata` and get rid of `mUpdateDirectory`? Is `mUpdateDirectory` just the directory version of this feature?

Reply:
done